scripts/mutect2_wdl/unsupported/mutect2_opt.wdl

```diff
@@ -134,82 +168,118 @@ workflow Mutect2 {
         gatk4_jar = gatk4_jar,
         gatk4_jar_override = gatk4_jar_override,
         m2_docker = m2_docker,
-        output_vcf_name = ProcessOptionalArguments.output_name
+        output_vcf_name = ProcessOptionalArguments.output_name,
+        disk_space_gb = merge_bamout_size
     }
   }
 
-  if (is_run_orientation_bias_filter) {
-      call CollectSequencingArtifactMetrics {
-        input:
-            preemptible_attempts = preemptible_attempts,
-            m2_docker = m2_docker,
-            tumor_bam = tumor_bam,
-            tumor_bam_index = tumor_bam_index,
-            ref_fasta = ref_fasta,
-            ref_fasta_index = ref_fasta_index,
-            picard_jar = picard_jar
+  if (is_run_orientation_bias_filter && !defined(tumor_sequencing_artifact_metrics)) {
+    Int seq_artifact_metrcis_size = tumor_bam_size + ref_size + disk_pad
+    call CollectSequencingArtifactMetrics {
+      input:
+        preemptible_attempts = preemptible_attempts,
+        m2_docker = m2_docker,
+        tumor_bam = tumor_bam,
+        tumor_bam_index = tumor_bam_index,
+        ref_fasta = ref_fasta,
+        ref_fasta_index = ref_fasta_index,
+        picard_jar = picard_jar,
+        disk_space_gb = seq_artifact_metrcis_size
       }
   }
 
+  # If contamination vcf is defined, run CalculateContamination
+  if (defined(variants_for_contamination)) {
+    Int calculate_contam_size = tumor_bam_size + ceil(size(variants_for_contamination, "GB") * 2) + disk_pad
```

Comment:
the output for this is a tiny file so instead of accounting for this by doubling the bam size, i just went with the size of the *hopefully* smaller contam vcf